torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -677,8 +677,9 @@ PjRtComputationClient::ExecuteReplicated(
   TF_VLOG(5) << "ExecuteReplicated acquiring PJRT device lock for "
              << spmd_device_str << " Done";
 
-  std::optional<std::vector<xla::PjRtFuture<xla::Status>>> returned_futures(
-      devices.size());
+  using FutureVector = std::vector<xla::PjRtFuture<xla::Status>>;
```

Comment:
Please remove the `FutureVector` alias. The triple-nested template is verbose, but I'd rather match [the API we're using here](https://github.com/openxla/xla/blob/13200274883ac910d5c25d5b82d50e98f42b1b39/xla/pjrt/pjrt_client.h#L1396).

Reply:
+1 I prefer sticking to non-aliased type names, unless it's used frequently.